src/renderer/qssWidget/js/qssOpenUsbWidget.js

```diff
@@ -0,0 +1,350 @@
+                mountUsbEvent.fire(messageChannel, openUSB.model.messages);
+                break;
+            case "Unmount":
+                // fires the event that ejects any attached USB drive
+                unmountUsbEvent.fire(messageChannel, openUSB.model.messages);
+                break;
+            default:
+                // do nothing in every other case
+                break;
+            }
+            openUSB.close(keyboardEvent);
+        }
+    };
+
+    /**
+     * A handler for the `repeater` instance in the QSS widget menu. Takes care of rendering
+     * a particular setting option and handling user interaction.
+     */
+    fluid.defaults("gpii.qssWidget.openUSB.presenter", {
+        gradeNames: ["fluid.viewComponent", "gpii.qssWidget.button"],
+        model: {
+            item: null
+        },
+        styles: {
+            active: "fl-qssWidgetMenu-active",
+            default: "fl-qssWidgetMenu-default"
+        },
+        modelListeners: {
+            item: {
+                this: "{that}.container",
+                method: "text",
+                args: ["{that}.model.item.value"]
+            },
+            "{repeater}.model.value": [{
+                funcName: "gpii.qssWidget.openUSB.presenter.animateActivation",
+                args: ["{change}.value", "{that}.model.item", "{that}.container", "{that}.options.styles"],
+                includeSource: "settingAlter"
+            }]
+        },
+        events: {
+            onItemFocus: "{repeater}.events.onItemFocus"
+        },
+        listeners: {
+            "onCreate.applyStyles": {
+                funcName: "gpii.qssWidget.openUSB.presenter.applyStyles",
+                args: ["{that}", "{that}.container", "{repeater}.model.styles"]
+            },
+            onItemFocus: {
+                funcName: "gpii.qssWidget.openUSB.presenter.focusItem",
+                args: [
+                    "{that}",
+                    "{focusManager}",
+                    "{that}.container",
+                    "{arguments}.0" // index
+                ]
+            }
+        },
+        invokers: {
+            activate: {
+                func: "{repeater}.updateValue",
+                args: [
+                    "{that}.model.item.key",
+                    "{arguments}.0" // keyboardEvent
+                ]
+            }
+        }
+    });
+
+    /**
+     * Focuses the current QSS menu option if its index matches the specified `index` parameter.
+     * @param {Component} that - The `gpii.qssWidget.menu.presenter` instance.
+     * @param {focusManager} focusManager - The `gpii.qss.focusManager` instance. for the QSS.
+     * @param {jQuery} container - A jQuery object representing the setting option's container.
+     * @param {Number} index - The index of the setting option to be focused.
+     */
+    gpii.qssWidget.openUSB.presenter.focusItem = function (that, focusManager, container, index) {
```

Comment:
More cut and paste from the other 2 locations - rewrite so this just appears once